example_experiments/transfer/asreader_wdw_sciq_mix_train_sciq_val.json

```diff
@@ -0,0 +1,30 @@
+// train a model on a combined dataset with SciQ train and a subset of WhoDidWhat train and see how well it does on SciQ.
```

Comment:
Your commit says you're "temporarily" adding these files back - did you mean to keep them in the PR?